src/microbe_stage/MicrobeStage.cs

```diff
@@ -279,9 +279,19 @@ public override void StartNewGame()
 
     public override void StartMusic()
     {
+        var biome = CurrentGame!.GameWorld.Map.CurrentPatch!.BiomeTemplate;
+
+        var context = new[] { MusicContext.General };
+
+        if (biome.InternalName == "aavolcanic_vent")
+            context[0] = MusicContext.PatchVents;
```

Comment:
If it's not too much work, I think it would be nice to let the biomes define for themselves which contexts they support in the json file instead of basing it off the name. That way, we don't have to come back and modify this script whenever we add a new biome or add new contexts for existing biomes.

Reply:
I think that's a great idea to reduce the hardcoding here, and I think I probably had the same idea in mind when thinking about this contexts feature years ago.